src/storage/recovery/recovery_manager.cpp

```diff
@@ -136,7 +136,7 @@ uint32_t RecoveryManager::ProcessDeferredTransactions(terrier::transaction::time
   uint64_t num_bytes = 0;
   for (auto it = deferred_txns_.begin(); it != upper_bound_it; it++) {
     {
-      common::ScopedTimer<std::chrono::microseconds> scoped_timer(&elapsed_us);
+      common::ScopedTimer<std::chrono::milliseconds> scoped_timer(&elapsed_us);
```

Comment:
If you're changing this to milliseconds, you need to update to code to be `elapsed_ms` instead of `elapsed_us`. Especially in the csv file where the column name is still `elapsed_us`